database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/action/DBRun.java

```diff
@@ -17,29 +17,28 @@
 package co.cask.hydrator.plugin.db.batch.action;
 
 import co.cask.hydrator.plugin.DBManager;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
 import java.sql.Connection;
 import java.sql.Driver;
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.util.Properties;
 
 /**
  * Class used by database action plugins to run database commands
  */
 public class DBRun {
   private final QueryConfig config;
   private final Class<? extends Driver> driverClass;
-  private static final Logger LOG = LoggerFactory.getLogger(DBRun.class);
 
   public DBRun(QueryConfig config, Class<? extends Driver> driverClass) {
     this.config = config;
     this.driverClass = driverClass;
   }
 
-  public void run() {
+  public void run() throws IOException, SQLException, InstantiationException, IllegalAccessException {
     DBManager dbManager = new DBManager(config);
 
     try {
```

Comment:
Not your PR but config can be null should check that and throw helpful message than NPE

Reply:
It won't actually be null when this code runs, because the constructor has a default, non-null value.